gulpfile.js

```diff
@@ -0,0 +1,39 @@
+var gulp = require('gulp');
+var sass = require('gulp-sass');
+var rename = require('gulp-rename');
+var cleanCSS = require('gulp-clean-css');
+var autoprefixer = require('gulp-autoprefixer');
+var browserSync = require('browser-sync').create();
+
+gulp.task('default', ['serve']);
```

Comment:
Interesting point. Actually, the default should be `sass` - since the source needs to be processed before minification. `minify-css` only takes the processed css and minifies it. Without `sass` running first, it would simply reminify the same old file. I broke up the two tasks to provide build streaming to both the css and min.css files simultaneously. I'll swap the defaults around & provide info on running browser-sync separately.